wasm-repl/Cargo.toml

```diff
@@ -14,4 +14,7 @@ nickel-lang-core = { workspace = true, default-features = false, features = ["re
 
 [lib]
 crate-type = ["cdylib", "rlib"]
-bench = false
\ No newline at end of file
+bench = false
+
+[features]
+nix-experimental = ["nickel-lang-core/nix-experimental"]
```

Comment:
I understand the other ones, but will nix really build for wasm?

Reply:
[this seems to build, so i guess so?](https://github.com/NixOS/nixpkgs/compare/master...KiaraGrouwstra:nixpkgs:nickel-enable-nix-import-wasm?expand=1) 🤷‍♀️🤷‍♀️

Reply:
Fair enough. It does fail, though, if I add `--target wasm32-unknown-unknown` and use that together with the "nix-experimental" feature.

Reply:
fair! i'll just remove it then